packages/cli/contracts/src/tables/Table1.sol

```diff
@@ -19,7 +19,7 @@ import { PackedCounter, PackedCounterLib } from "@latticexyz/store/src/PackedCou
 // Import user types
 import { Enum1, Enum2 } from "./../types.sol";
 
-uint256 constant _tableId = uint256(keccak256("/Table1"));
+uint256 constant _tableId = uint256(bytes32(abi.encodePacked(bytes16(""), bytes16("Table1"))));
```

Comment:
does `abi.encodePacked` leave padding for an empty `bytes16`?

Reply:
also are we able to make library calls when building these constants? it would be nice to do like
```
uint256 constant _tableId = TableId.from("", "Table1");
```

Reply:
> does abi.encodePacked leave padding for an empty bytes16?

yes, `bytes16` always takes 16 bytes in `abi.encodePacked`, independent of its content

> also are we able to make library calls when building these constants?

Unfortunately not - we're lucky `abi.encodePacked` is an exception that is considered "constant", we can't even do bitshifts in constants

> and if not, maybe a uint type alias?

Like `type TableId is uint256`? I _think_ we could do that. Can you elaborate on why you'd prefer that over just `uint256`?

Reply:
I guess the type alias wouldn't matter if we can't call methods on that alias. So it's not much better.

Reply:
The slightly annoying thing with a custom `TableId` type is `uint256` can't be implicitly converted to `TableId`, so we'd have to do `TableId.wrap(tableId)` in all consuming libraries that call any Store function. For simplicity my preference would be to keep it `uint256`.

Reply:
Okay think I found a good compromise: with a library `TableId` and `using TableId for uint256` in `StoreCore` we can do `tableId.toString()`, without the overhead of a custom type

Reply:
Would this affect all uints? Wondering if we should be more explicit like castToString.

Reply:
this means inside of `StoreCore` you could call the `toString` method on all `uint256`, yes. But as long as we don't do that it should not be a problem. We can also just use `TableId.toString` and remove using for if you think it's an issue.

Reply:
I've just personally used `toString` on `uint` specifically to translate e.g. `123` into `"123"`. Same is true for a couple of common Solidity libraries: https://github.com/transmissions11/solmate/blob/main/src/utils/LibString.sol#L4-L8

So I'm just a little wary of overlapping naming. I'll personally need to make a mental leap to remind myself that this is more of a cast operation.